ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/EstablishedPeers.hs

```diff
@@ -247,12 +249,14 @@ jobPromoteColdPeer PeerSelectionActions {
         let (failCount, knownPeers') = KnownPeers.incrementFailCount
                                          peeraddr
                                          (knownPeers st)
+            (fuzz, fuzzRng') = randomR (-2, 2 :: Double) fuzzRng
```

Comment:
Range doesn't match comment. Update the comment to say -2 to 2,

Reply:
done